app/src/main/java/com/pwhs/quickmem/presentation/app/classes/add_folder/AddFolderToClassScreen.kt

```
@@ -0,0 +1,84 @@
+        }
+        LoadingOverlay(
+            isLoading = isLoading
+        )
+    }
+}
+
+@Preview(showBackground = true, showSystemUi = true)
+@Composable
+fun AddFolderUIPreview() {
+
+}
```

detekt

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)